syntax = "proto3";

package Protocol;

message UpgradeEquipItem{
	uint64 SlotIndex = 1;
}